includes/IRenderComponent.h

```diff
@@ -24,6 +24,7 @@ class IRenderComponent : public IComponent {
 public:
     virtual void render() = 0;
     virtual void renderShadow(std::shared_ptr<ShaderProgram> &shaderProgram) = 0;
+    virtual void renderEmissive(std::shared_ptr<ShaderProgram> &shaderProgram) = 0;
```

Comment:
We should have some kind of documentation here as well. Since it will be overridden usage is hard to describe, but I think it would be possible to describe assumptions of the ShaderProgram etc.